go.mod

```diff
@@ -75,6 +76,7 @@ replace (
 	github.com/containerd/containerd => ./.empty-mod/
 	github.com/gogo/googleapis => github.com/gogo/googleapis v1.3.2
 	github.com/golang/protobuf => github.com/golang/protobuf v1.3.5
+	github.com/intel/goresctrl => github.com/askervin/goresctrl v0.0.0-20210416130319-f4c2f987ba1f
```

Comment:
Is this PR WIP?

Reply:
@AkihiroSuda , thanks, good catch! This PR was indeed still using a development version of goresctrl blockio API.

This PR is no more WIP, it's good for review now. (Though I was already kind of hoping it was... :) )